src/libs/DateUtils.ts

```diff
@@ -266,7 +266,7 @@ function formatToLongDateWithWeekday(datetime: string | Date): string {
  *
  * @returns Sunday
  */
-function formatToDayOfWeek(datetime: string | Date): string {
+function formatToDayOfWeek(datetime: Date): string {
     return format(new Date(datetime), CONST.DATE.WEEKDAY_TIME_FORMAT);
```

Comment:
```suggestion
    return format(datetime, CONST.DATE.WEEKDAY_TIME_FORMAT);
```

It it's going to be always `Date` we don't need to create it again, right?